### PR TITLE
Adds Malfunctioning AI voice emulation module

### PR DESCRIPTION
**Pull request by Kubisopplay**
## About The Pull Request
Adds a fully functional voicechanger to AI. It lets it pick voice span, verb, and the apparent name. Since i am terrible at naming and designing UIs this PR is a cry for help. Also, i have a permission to do it from Kyler.

## Why It's Good For The Game
Lets malfies topple the station with roleplay instead of white-hot plasmafire. Or lets them troll people mercilessly, without any outward signs of malf, whatever they want.

## Changelog

:cl:
expansion: Lets malfunctioning AIs upgrade their voice synths, so they can impersonate anyone.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/traitor/equipment/Malf_Modules.dm [hunk 1]
```diff
@@ -851,5 +851,119 @@ GLOBAL_LIST_INIT(malf_modules, subtypesof(/datum/ai_module))
...
+    ui.open()
+
+
+/obj/machinery/ai_voicechanger/ui_data(mob/user)
+  var/list/data = list()
+  data["voices"] = list() //manually adding this since i dont see other option
+  data["voices"]+=list("Normal")
+  data["voices"]+=list(SPAN_ROBOT)
+  data["voices"]+=list(SPAN_YELL)
+  data["voices"]+=list(SPAN_CLOWN)
```
Comment (Fikou) [inline code/modules/antagonists/traitor/equipment/Malf_Modules.dm:916]: why not just start that list with those instead of adding them 1 by 1

Comment (Kubisopplay) [reply code/modules/antagonists/traitor/equipment/Malf_Modules.dm:916]: I have been infected with catbrain disease. Will fix it

File: code/modules/antagonists/traitor/equipment/Malf_Modules.dm [hunk 2]
```diff
@@ -851,5 +851,119 @@ GLOBAL_LIST_INIT(malf_modules, subtypesof(/datum/ai_module))
...
 
+
+
+/datum/ai_module/upgrade/voice_changer
+	name = "Voice Changer"
+	description = "Allows you to change the AI's voice. Upgrade is active immediately upon purchase."
+	cost = 40
+	one_purchase = TRUE
+	power_type = /datum/action/innate/ai/voice_changer
+	unlock_text = "<span class='notice'>OTA firmware distribution complete! Voice changer online.</span>"
```
Comment (Kylerace) [inline code/modules/antagonists/traitor/equipment/Malf_Modules.dm:862]: use span_notice()

**Review by Kylerace — CHANGES_REQUESTED** [review]

**Comment by Kubisopplay** [comment]
Oh god i made a big mistake while using git

**Comment by Kubisopplay** [comment]
I am sorry for being retarded

**Pull request closed without merging**